Assignment1/pages/game.vue

```diff
@@ -0,0 +1,20 @@
+   <template>
+    <v-card variant="tonal">
+      <v-card-title>Hello Welcome to Mystery Number Madness!</v-card-title>
+      <v-card-text><p>Welcome to my fun filled mystery number game! You will be tasked with figuring out my number and will get fun feedback along the way! Guess the mystery number between 1 and 100 and most importantly, HAVE FUN!</p></v-card-text>
```

Comment:
```suggestion
      <v-card-text>
        Welcome to my fun filled mystery number game! You will be tasked with 
        figuring out my number and will get fun feedback along the way! Guess 
        the mystery number between 1 and 100 and most importantly, HAVE FUN!
      </v-card-text>
```
Since this text is the only content in here, wrapping in a `<p>` element provides no value. Vuetify resets a lot the default element styling as well, so a `<p>` ends up acting like a regular `<div>`.

Also, it is common to drop the content and closing element tag down vertically. The reason for this is because as you read through the template, you gain a tree view of how your elements are nested through their indentation and can easily identify what starts/ends where.